### PR TITLE
SDK 2.5 integration

### PR DESCRIPTION
**Pull request by kubaau**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
   workflow_dispatch:
 
 env:
-  CPP_SDK_TAG: "2.4.3"
+  CPP_SDK_TAG: "2.5.0-beta.2"
```
Comment (djova-dolby) [inline .github/workflows/build.yml:8]: Lets just wait for beta.3 with the memory leak fix, otherwise LGTM.

**Review by djova-dolby — DISMISSED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -35,9 +35,9 @@ jobs:
           LIB_PATH_X86: "${{ github.workspace }}/${{ env.CPP_SDK_DESTINATION }}/sdk-release-x86/lib"
         run: |
           echo "Removing symlinks from ${LIB_PATH_ARM}"
-          rm ${LIB_PATH_ARM}/libav*.dylib
+          find ${LIB_PATH_ARM} -type l -delete
```
Comment (Adam1-3) [inline .github/workflows/build.yml:38]: nice

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -97,3 +97,41 @@ jobs:
...
+      fail-fast: false
+      matrix:
+        version: ["4.27", "5.2"]
+    runs-on: ["self-hosted", "X64", "Windows"]
+    env:
+      BUILD_PLATFORM: 'linux64'
+    steps:
+      - name: Code checkout
+        uses: actions/checkout@v3
+      - name: Download C++ SDK release asset for Windows
```
Comment (Adam1-3) [inline .github/workflows/build.yml:113]: not linux ;)

File: DolbyIO/Source/DolbyIO.Build.cs [hunk 4]
```diff
@@ -5,57 +5,77 @@
...
+                Path.Combine(LibDir, "dolbyio_comms_sdk.lib"),
+            });
+
+            string[] Dlls = new string[] { "avutil-57.dll", "avcodec-59.dll", "dvclient.dll", "dolbyio_comms_media.dll", "dolbyio_comms_sdk.dll" };
+            PublicDelayLoadDLLs.AddRange(Dlls);
+
+            string BinDir = Path.Combine(SdkDir, "bin");
+            foreach (string Dll in Dlls)
+            {
+                RuntimeDependencies.Add(Path.Combine(BinDir, Dll));
```
Comment (Adam1-3) [inline DolbyIO/Source/DolbyIO.Build.cs:40]: Could we use AddRange() here?

Comment (kubaau) [reply DolbyIO/Source/DolbyIO.Build.cs:40]: No, RuntimeDependecies does not have AddRange.

File: DolbyIO/Source/Private/DolbyIOConversions.cpp [hunk 5]
```diff
@@ -193,19 +193,55 @@ namespace DolbyIO
...
+				Ret.resolution = screen_share_content_info::max_resolution::actual_captured;
 		}
+
+		switch (DownscaleQuality)
+		{
+			case EDolbyIOScreenshareDownscaleQuality::Medium:
+				Ret.quality = screen_share_content_info::downscale_quality::medium;
+				break;
+			case EDolbyIOScreenshareDownscaleQuality::High:
+				Ret.quality = screen_share_content_info::downscale_quality::highest;
```
Comment (Adam1-3) [inline DolbyIO/Source/Private/DolbyIOConversions.cpp:235]: Typo?

File: DolbyIO/Source/Private/DolbyIOConversions.cpp [hunk 6]
```diff
@@ -193,19 +193,55 @@ namespace DolbyIO
...
+				break;
+			case EDolbyIOScreenshareDownscaleQuality::Highest:
+				Ret.quality = screen_share_content_info::downscale_quality::high;
```
Comment (Adam1-3) [inline DolbyIO/Source/Private/DolbyIOConversions.cpp:238]: and here?

**Review by Adam1-3 — APPROVED** [review]

**Pull request merged**